tests/acceptance/deploy/k8s/apps_dns.yaml

```diff
@@ -0,0 +1,135 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: apps-dns
+  namespace: tests
+  labels:
+    app: apps-dns
+spec:
+  clusterIP: "10.43.100.100"
+  ports:
+  - name: dns
+    protocol: UDP
+    port: 53
+    targetPort: dns
+  selector:
+    app: apps-dns
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: apps-dns
+  namespace: tests
+  labels:
+    app: apps-dns
+data:
+  Corefile: |-
+    . {
+      errors
+      health
+
+      svcdiscovery {
+        tls_ca_path /tls/ca.pem
+        tls_client_cert_path /tls/cert.pem
+        tls_client_key_path /tls/key.pem
+        sdc_host service-discovery-controller.tests.svc
+        sdc_port 8054
+        ttl 300
+      }
+
+      forward . /config/forward.conf
+
+      loop
+      reload
+      loadbalance
+    }
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: apps-dns
+  namespace: tests
+  labels:
+    app: apps-dns
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: apps-dns
+  template:
+    metadata:
+      labels:
+        app: apps-dns
+    spec:
+      volumes:
+      - name: corefile-config
+        configMap:
+          name: apps-dns
+      - name: forward-config
+        emptyDir: {}
+      - name: client-tls
+        secret:
+          secretName: var-cf-app-sd-client-tls
+          items:
+          - key: ca
+            path: ca.pem
+          - key: certificate
+            path: cert.pem
+          - key: private_key
+            path: key.pem
+      initContainers:
+      - name: resolv-writer
+        image: kubecf-apps-dns:latest
+        imagePullPolicy: Never
+        command: [/resolvwriter]
```

Comment:
Shouldn't this be baked into the image?

Reply:
No. The image has 2 binaries. The entrypoint is already coredns (which is the correct one):
https://github.com/SUSE/kubecf-apps-dns/blob/2175967e311c3ab8b4149129443caebda160374a/image/Dockerfile#L38